packages/mobile/src/screens/trending-screen/TrendingFilterButton.tsx

```diff
@@ -14,14 +24,22 @@ const useStyles = makeStyles(({ spacing, palette }) => ({
     textTransform: 'none'
   }
 }))
+
 export const TrendingFilterButton = () => {
   const styles = useStyles()
+  const dispatchWeb = useDispatchWeb()
+  const trendingGenre = useSelectorWeb(getTrendingGenre) ?? Genre.ALL
+
+  const handlePress = useCallback(() => {
+    dispatchWeb(setVisibility({ modal: MODAL_NAME, visible: true }))
+  }, [dispatchWeb])
+
   return (
     <Button
```

Comment:
Should we just use `HeaderButton` here?

Reply:
yes will update

Reply:
btw including this in another pr that will do work merging screen-header, header, and header-button